oxidation/client/android/app/src/main/res/xml/config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
   <access origin="*" />
-
-
-</widget>
+  
+  
+</widget>
```

Comment:
missing newline

did you configure [editorconfig](https://editorconfig.org/) ?

Reply:
It's a generated file, we should use a precommit to fix it automaticaly

Reply:
https://github.com/Scille/parsec-cloud/issues/2884